rest/src/db/CatapultDb.js

```diff
@@ -304,24 +305,31 @@ class CatapultDb {
 	 * `pageSize` and `pageNumber`. 'sortField' must be within allowed 'sortingOptions'.
 	 * @returns {Promise.<object>} Transactions page.
 	 */
-	transactions(group, filters, options) {
+	async transactions(group, filters, options) {
 		const sortingOptions = { id: '_id' };
 
-		const buildAccountConditions = () => {
-			if (undefined !== filters.address)
-				return { 'meta.addresses': Buffer.from(filters.address) };
+		const buildAccountConditions = async () => {
+			// Check multisig graph if address is used in search criteria for cosigning,
+			// Then, show transactions for other cosigers.
+			if (undefined !== filters.address) {
```

Comment:
Whats the perf overhead of making this call for non-multisig accounts?

Reply:
I think it's fine. the extra db call is only done when the address filter is provided and the accountAddress field is indexed in multisigs

Reply:
How do you the account is multisig or not without checking the db?